FitnessTracker/Contracts/ApiRoutes.cs

```diff
@@ -14,28 +12,29 @@ public static class Example
             public const string Update = Base + "example/{exampleId}";
             public const string Delete = Base + "example/{exampleId}";
         }
-        
+
         public static class User
         {
             public const string GetAll = Base + "user/all";
             public const string Get = Base + "user";
             public const string Update = Base + "user";
             public const string UpdateRole = Base + "user/{userId}";
             public const string Delete = Base + "user/{userId}";
+            public const string Test = Base + "user/Test";
```

Comment:
ok generalnie z grubsza mam przejrzane wszystko, tylko tego jednego endpointa nie rozumiem i nie widzę nigdzie indziej wzmiankowanego

Reply:
ok, to zakładam że zostanie ten endpoint wywalony i zamykam.